tests/erlang_tests/small_big_ext.erl

```diff
@@ -55,44 +55,64 @@ start() ->
     true = test_reverse(pow(59) - 1, <<131, 110, 8, 0, 255, 255, 255, 255, 255, 255, 255, 7>>),
     true = test_reverse(-pow(59), <<131, 110, 8, 1, 0, 0, 0, 0, 0, 0, 0, 8>>),
 
+    % TODO: enable as soon as serialization for big integers is ready
+    %true = test_reverse(
+    %    erlang:binary_to_integer(?MODULE:id(<<"8000000000000001">>), 16),
+    %    <<131, 110, 8, 0, 1, 0, 0, 0, 0, 0, 0, 128>>
+    %),
+    %true = test_reverse(
+    %    erlang:binary_to_integer(?MODULE:id(<<"-8000000000000002">>), 16),
+    %    <<131, 110, 8, 1, 2, 0, 0, 0, 0, 0, 0, 128>>
+    %),
+    %true = test_reverse(
+    %    erlang:binary_to_integer(?MODULE:id(<<"100000000000000000000000000000000">>), 16),
+    %    <<131, 110, 17, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1>>
+    %),
+
     %% missing sign
```

Comment:
We are not testing INT256 edge cases, and these tests don't pass on AtomVM but pass on BEAM:

```
    true = test_reverse(pow(255) - 1, <<131,110,32,0,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,127>>),
    true = test_reverse(pow(255), <<131,110,32,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,128>>),
    true = test_reverse(-pow(255), <<131,110,32,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,128>>),
    true = test_reverse(-pow(255) - 1, <<131,110,32,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,128>>),
```

Reply:
I didn't yet implement most of the other arithmetic operations, such as `-/2`. As soon as the general approach works I'll repeat it for all the missing operations.
I was focused on the `*/2` operation.